systemd/ipconfig.sh

```diff
@@ -0,0 +1,3 @@
+IP=`ip address show dev eth0 | grep inet | head -n1 | cut -d ' ' -f 6 | cut -d '/' -f 1`
+echo $IP
+sed -i "s/0.0.0.0:21337/$IP:21337/" witnet.toml
```

Comment:
This may accidentally insert your internal IP in the `public_addr` field too, right?

Reply:
That's why I've checked with Contabo and Digital Ocean servers and it works there. Of course it won't work everywhere. But for a first version I wanted to try something else than relying on external service. As long as it works for the VPS it covers most of the users.